internal/versions/api.go

```diff
@@ -104,6 +104,14 @@ func (h *apiHandler) V2ListSupportedOpenshiftVersions(ctx context.Context, param
 				continue
 			}
 
+			// In order to handle multi-arch release images correctly in the UI, we need to tune
+			// their DisplayName to contain an appropriate suffix. If the suffix comes from the
+			// JSON definition we do nothing, but in case it's missing there (because CVO is not
+			// reporting it), we add it ourselves.
+			if len(supportedArchs) > 1 && !strings.HasSuffix(*releaseImage.Version, "-multi") {
+				releaseImage.Version = swag.String(*releaseImage.Version + "-multi")
```

Comment:
Can you assign the value to a new var, e.g. 'displayName', and use it in line 120 (no need to change releaseImage).

Reply:
Of course that makes sense